artifactory/services/utils/searchutil.go

```diff
@@ -189,6 +190,7 @@ func SearchBySpecWithPattern(specFile *CommonParams, flags CommonConf, requiredA
 
 // Use this function when running Aql with pattern
 func SearchBySpecWithAql(specFile *CommonParams, flags CommonConf, requiredArtifactProps RequiredArtifactProps) (reader *content.ContentReader, err error) {
+	log.Info("Searching for items in Artifactory...")
```

Comment:
@sverdlov93 what is the reason for adding this log message?

This change is causing lot of log noise and causing lot of pain when scrolling log output from tools that consumed this api. cc @ns-dsinghal

Reply:
Hi @ns-kbhat @ns-dsinghal , the log has been removed.